p2p/nodestate/nodestate.go

```diff
@@ -723,7 +784,10 @@ func (ns *NodeStateMachine) addTimeout(n *enode.Node, mask bitMask, timeout time
 	ns.removeTimeouts(node, mask)
 	t := &nodeStateTimeout{mask: mask}
 	t.timer = ns.clock.AfterFunc(timeout, func() {
-		ns.SetState(n, Flags{}, Flags{mask: t.mask, setup: ns.setup}, 0)
+		if debugPrints {
+			fmt.Println("timeout", n.ID(), Flags{mask, ns.setup})
+		}
```

Comment:
What's all this about `debugPrints` and raw `fmt.Println` ? It's not the way we do logging in go-ethereum...?

Reply:
This was just my own quick debug stuff, I forgot to remove it in the final version.